ocaml/src/capabilities.ml

```diff
@@ -0,0 +1,54 @@
+(*
+Copyright (C) 2016 iNuron NV
+
+This file is part of Open vStorage Open Source Edition (OSE), as available from
+
+
+    http://www.openvstorage.org and
+    http://www.openvstorage.com.
+
+This file is free software; you can redistribute it and/or modify it
+under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+as published by the Free Software Foundation, in version 3 as it comes
+in the <LICENSE.txt> file of the Open vStorage OSE distribution.
+
+Open vStorage is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY of any kind.
+ *)
+
+
+
+module OsdCapabilities = struct
```

Comment:
Ironically, they were first called AsdCapabilities, but I changed my mind. I must admit the difference between Asd and Osd is no longer that clear to me. 
Osd started out as the common abstract interface that both Kinetic and Asds supported, but that's not really the case anymore.

The concept should allow us to remove the various tests we do to see if individual operations are supported or not( [for example here](https://github.com/openvstorage/alba/blob/master/ocaml /src/asd_client.ml#L134) )